doc/databinding/@bind.md

```diff
@@ -90,13 +90,29 @@ model: Model;
 
 For details see [`@bindAll`](./@bindAll.md).
 
+### config.type
+
+A [type](./@property.md#configtype) may be given to enforce type checks in JavaScript.
+
 ### config.typeGuard
 
-A [`typeGuard`](./@property.md#configtypeguard) may be given to perform value checks.
+A [type guard](./@property.md#configtypeguard) may be given to perform value checks.
 
-### config.type
+### config.default
+
+The [default](./@property.md#configdefault) value of the property.
+
+### config.nullable
+
+Whether or not the value is  [nullable](./@property.md#confignullable). Is `true` by default.
+
+### config.equals
+
+How the property determines whether an incoming value [equals](./@property.md#configequals) the current one.
+
+### config.convert
```

Comment:
Probably there aren't, but are there any wide-spread conventions/libraries for type conversions that would make sense to be inspired from for the 'auto' conversion behavior?

Reply:
no idea